extensions/interactions/CodeRepl/CodeReplRulesServiceSpec.js

```diff
@@ -148,6 +148,27 @@ describe('Code REPL rules service', function() {
     });
   });
 
+  describe('\'output contains\' rule', function() {
+      var RULE_INPUT = {
+        x: '1'
+      };
+
+      it('should check if output contains some content', function() {
```

Comment:
Well, more like: is 'a b' contained in 'a    b', is '    b' contained in 'b', is 'a\nb' contained in 'a\n\nb', etc.

Basically, see if you can come up with a few trickier tests that characterizes how OutputContains() behaves. To put it another way, it would be nice if someone else can deduce from these tests how OutputContains() is meant to behave in the general case, even if they haven't seen its implementation/documentation. At the very least it would be nice to check something where the normalization of the strings actually matters.